sdk/src/main/scala/io/horizen/account/state/ForgerStakeStorage.scala

```diff
@@ -348,6 +348,7 @@ object ForgerStakeStorageV2 extends ForgerStakeStorage {
 object ForgerStakeStorage {
```

Comment:
At line 343 above: `forgerStakeArray.getSize(view) <= 5000`
Consider using a constant definition, also given that the same value is used in V2 too.

Reply:
change not neded but let's check  the 5000 is no more present in V2